MinimedKit/Extensions/NSData.swift

```diff
@@ -71,6 +71,14 @@ extension Data {
      */
 }
 
+extension Data {
+    func reverseBytes() -> Data {
```

Comment:
I'm having trouble figuring out what to do with the data structure that reversed() gives me. It's a ReversedRandomAccessCollection<Data>, and it looks like I have to carry that all the way through to each GlucoseEvent. Can you help me understand how to appropriately use reversed() on the page Data?